### PR TITLE
catamaran: init at 0-unstable-2024-03-02

### LINKED ISSUE #291875: Package request: catamaran-sans font
**Project description**

Catamaran sans is a font for tamil language 

**Metadata**

* homepage URL: https://fonts.google.com/specimen/Catamaran
* source URL: https://github.com/google/fonts/tree/main/ofl/catamaran
* license: OFL
* platforms: ?

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by ghost**
## Description of changes

I have added a new font package called catamaran. This closes #291875.

GitHub page: https://fonts.google.com/specimen/Catamaran

Source used: https://www.1001fonts.com/download/catamaran.zip

Thank you, @JulienMalka, for letting me know what I did wrong in #292765. I sincerely apologize to anyone I may have accidentally pinged.

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/data/fonts/catamaran/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "catamaran";
+  version = "2024-03-02";
```
Comment (lolbinarycat) [inline pkgs/data/fonts/catamaran/default.nix:8]:
```suggestion
  version = "0-unstable-2024-03-02";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

**Review by ghost — COMMENTED** [review]
Fixed the version

**Review by lolbinarycat — APPROVED** [review]
no syntax or hygine issues as far as i can see, no binary files to test either.

**Review by h7x4 — APPROVED** [review]
Everything looks good, only squashing the commits was left. Feel free to ping a committer when it takes so long to get the PR merged.

**Pull request merged**